src/test/java/io/cdap/plugin/gcp/bigquery/action/BigQueryExecuteTest.java

```diff
@@ -228,15 +230,18 @@ public void testValidateRetryConfigurationWithInvalidReadTimeout() {
 
   @Test
   public void testValidateWritePreferenceWithInvalidValue() {
+    Set<String> validPreferences = ImmutableSet.of(
+        JobInfo.WriteDisposition.WRITE_EMPTY.name(),
+        JobInfo.WriteDisposition.WRITE_APPEND.name(),
+        JobInfo.WriteDisposition.WRITE_TRUNCATE.name()
```

Comment:
make it a static final variable in `BQExecute.Config` class

Reply:
we can also do something like this so that we don't need to maintain static list.

```
Set<String> validPreferences = Arrays.stream(JobInfo.WriteDisposition.values()).map(Enum::name).collect(Collectors.toSet());
```